kaggle_environments/main.py

```diff
@@ -53,7 +53,12 @@
 parser.add_argument(
     "--render",
     type=json.loads,
-    help="Response from run, step, or load. Calls environment render. (default={mode='json'})",
+    help="Response from run, step, or load. Calls environment render (default={mode='json'}).",
+)
+parser.add_argument(
+    "--display",
```

Comment:
Currently to export html you have to pass in the command line argument `--render '{"mode":"html"}'` which feels arduous. This simplifies it to `--display html`.